config/nightwatch-globals-sauce.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
There is also this directory that has nightwatch stuff in it: `test/util/nightwatch-commands` although a bunch of those are helper functions. Wondering if anyone has strong opinions about whether these should live in a single place.